Wiki-Contribute-Fork.md

```diff
@@ -0,0 +1,79 @@
+# How to get a fork of our wiki and keep it syncronized
+
+These steps are standard for contributing not only to our wiki but any other project once you understand what is going on.
+
+## Steps to Follow
+
+### 1\. **Fork it** :fork_and_knife:
+
+You can get your own fork/copy of [FreeCodeCamp wiki](https://github.com/FreeCodeCamp/wiki) by using the
+
+[<kbd>
+  <b>Fork</b>
+</kbd>](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box)
+
+ button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).
```

Comment:
@Rafase282 they're in multiple lines so they appear as 

----
You can get your own fork/copy of [FreeCodeCamp wiki](https://github.com/FreeCodeCamp/wiki) by using the

[<kbd>
  <b>Fork</b>
</kbd>](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box)

 button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).

----
### instead of

----
You can get your own fork/copy of [FreeCodeCamp wiki](https://github.com/FreeCodeCamp/wiki) by using the <a href="https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box"><kbd><b>Fork</b></kbd></a> button or clicking [this](https://github.com/FreeCodeCamp/wiki/new/master?readme=1#fork-destination-box).

----

Notice the multiple lines due to the inserted line breaks.